integration/spark2/src/main/scala/org/apache/spark/sql/execution/strategy/DDLStrategy.scala

```diff
@@ -156,11 +156,22 @@ class DDLStrategy(sparkSession: SparkSession) extends SparkStrategy {
       case AlterTableSetPropertiesCommand(tableName, properties, isView)
         if CarbonEnv.getInstance(sparkSession).carbonMetastore
           .tableExists(tableName)(sparkSession) => {
+        val property = properties.find(_._1.equalsIgnoreCase("streaming"))
+        if (property.isDefined) {
+          if (!property.get._2.trim.equalsIgnoreCase("true")) {
+            throw new MalformedCarbonCommandException(
+              "Unsupported alter table to disable streaming property")
```

Comment:
Suggest change to `Streaming property can not be changed to 'false' once it is 'true'`

Reply:
And please add a TODO to mark we will remove this limitation later

Reply:
fixed